src/osx/webview_webkit.mm

```diff
@@ -123,6 +123,12 @@ - (id)initWithWxWindow: (wxWebViewWebKit*)inWindow;
 
     NSRect r = wxOSXGetFrameForControl( this, pos , size ) ;
     WKWebViewConfiguration* webViewConfig = [[WKWebViewConfiguration alloc] init];
+
+    // WebKit API available since macOS 10.11 and iOS 9.0
+    SEL fullScreenSelector = @selector(_setFullScreenEnabled:);
+    if ([webViewConfig.preferences respondsToSelector:fullScreenSelector])
+        [webViewConfig.preferences performSelector:fullScreenSelector withObject:[NSNumber numberWithBool:YES]];
```

Comment:
Very minor, but do we have to use `performSelector:` explicitly here? I thought that the compiler was smart enough to not warn about "possibly unhandled selector" after a `respondsToSelector:` check.

Reply:
We don't have to, but I found less error prone than to repeat the `@selector`.